docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/DataViewEnumerable.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    /// <summary>
+    /// Sample class showing how to use ShuffleRows.
+    /// </summary>
+    public static class DataViewEnumerable
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<DatasetUtils.SampleTemperatureData> enumerableOfData = DatasetUtils.GetSampleTemperatureData(5);
+
+            // Read in into an IDataView using ReadFromEnumerable. 
+            IDataView data = mlContext.Data.ReadFromEnumerable(enumerableOfData);
+
+            // We can now examine the records in the IDataView. We first create an enumerable of rows in the IDataView.
+            var rowEnumerable = mlContext.Data.CreateEnumerable<DatasetUtils.SampleTemperatureData>(data, true);
```

Comment:
Can we also demonstrate the use of the 3rd argument, schemaDefinition? That argument is rarely used and tested.

Reply:
Sounds good will do that!